tests/integration/scenarios/in_http_max_connections/tests/test_in_http_max_connections_001.py

```
@@ -93,3 +94,101 @@ def test_in_http_max_connections_blocks_and_recovers():
+        held_connection.sendall(b"POST / HTTP/1.1\r\n")
+
+        overflow_rejected = False
+        try:
+            response = run_curl_request(
+                f"http://127.0.0.1:{service.flb_listener_port}/",
+                payload='{"message":"idle-timeout-blocked"}',
+                headers=["Content-Type: application/json"],
+                http_mode="http1.1",
+            )
+            overflow_rejected = response["status_code"] != 201
+        except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)